[2017-07-21 09:30:00.752] [ERROR] dateFileLog - ReferenceError: SatTimeWarningAID is not defined
    at success (C:\NB\tm.FGEMS\bizServer\biz\auto\AlarmOutputLower.js:51:53)
    at Object.exports.safeSuccess (C:\NB\yujiang.Foil.Node\src\yjError.js:98:13)
    at Object.exports.handleResult (C:\NB\yujiang.Foil.Node\src\yjError.js:154:12)
    at Query._callback (C:\NB\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:202:38)
    at Query.Sequence.end (C:\NB\node_modules\mysql\lib\protocol\sequences\Sequence.js:96:24)
    at Query._handleFinalResultPacket (C:\NB\node_modules\mysql\lib\protocol\sequences\Query.js:143:8)
    at Query.EofPacket (C:\NB\node_modules\mysql\lib\protocol\sequences\Query.js:127:8)
    at Protocol._parsePacket (C:\NB\node_modules\mysql\lib\protocol\Protocol.js:271:23)
    at Parser.write (C:\NB\node_modules\mysql\lib\protocol\Parser.js:77:12)
    at Protocol.write (C:\NB\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (C:\NB\node_modules\mysql\lib\Connection.js:82:28)
[2017-07-21 09:32:00.749] [ERROR] dateFileLog - ReferenceError: SatTimeWarningAID is not defined
    at success (C:\NB\tm.FGEMS\bizServer\biz\auto\AlarmOutputLower.js:51:53)
    at Object.exports.safeSuccess (C:\NB\yujiang.Foil.Node\src\yjError.js:98:13)
    at Object.exports.handleResult (C:\NB\yujiang.Foil.Node\src\yjError.js:154:12)
    at Query._callback (C:\NB\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:202:38)
    at Query.Sequence.end (C:\NB\node_modules\mysql\lib\protocol\sequences\Sequence.js:96:24)
    at Query._handleFinalResultPacket (C:\NB\node_modules\mysql\lib\protocol\sequences\Query.js:143:8)
    at Query.EofPacket (C:\NB\node_modules\mysql\lib\protocol\sequences\Query.js:127:8)
    at Protocol._parsePacket (C:\NB\node_modules\mysql\lib\protocol\Protocol.js:271:23)
    at Parser.write (C:\NB\node_modules\mysql\lib\protocol\Parser.js:77:12)
    at Protocol.write (C:\NB\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (C:\NB\node_modules\mysql\lib\Connection.js:82:28)
[2017-07-21 09:40:00.751] [ERROR] dateFileLog - ReferenceError: SatTimeWarningAID is not defined
    at success (C:\NB\tm.FGEMS\bizServer\biz\auto\AlarmOutputLower.js:51:53)
    at Object.exports.safeSuccess (C:\NB\yujiang.Foil.Node\src\yjError.js:98:13)
    at Object.exports.handleResult (C:\NB\yujiang.Foil.Node\src\yjError.js:154:12)
    at Query._callback (C:\NB\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:202:38)
    at Query.Sequence.end (C:\NB\node_modules\mysql\lib\protocol\sequences\Sequence.js:96:24)
    at Query._handleFinalResultPacket (C:\NB\node_modules\mysql\lib\protocol\sequences\Query.js:143:8)
    at Query.EofPacket (C:\NB\node_modules\mysql\lib\protocol\sequences\Query.js:127:8)
    at Protocol._parsePacket (C:\NB\node_modules\mysql\lib\protocol\Protocol.js:271:23)
    at Parser.write (C:\NB\node_modules\mysql\lib\protocol\Parser.js:77:12)
    at Protocol.write (C:\NB\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (C:\NB\node_modules\mysql\lib\Connection.js:82:28)
[2017-07-21 09:50:00.748] [ERROR] dateFileLog - ReferenceError: SatTimeWarningAID is not defined
    at success (C:\NB\tm.FGEMS\bizServer\biz\auto\AlarmOutputLower.js:51:53)
    at Object.exports.safeSuccess (C:\NB\yujiang.Foil.Node\src\yjError.js:98:13)
    at Object.exports.handleResult (C:\NB\yujiang.Foil.Node\src\yjError.js:154:12)
    at Query._callback (C:\NB\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:202:38)
    at Query.Sequence.end (C:\NB\node_modules\mysql\lib\protocol\sequences\Sequence.js:96:24)
    at Query._handleFinalResultPacket (C:\NB\node_modules\mysql\lib\protocol\sequences\Query.js:143:8)
    at Query.EofPacket (C:\NB\node_modules\mysql\lib\protocol\sequences\Query.js:127:8)
    at Protocol._parsePacket (C:\NB\node_modules\mysql\lib\protocol\Protocol.js:271:23)
    at Parser.write (C:\NB\node_modules\mysql\lib\protocol\Parser.js:77:12)
    at Protocol.write (C:\NB\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (C:\NB\node_modules\mysql\lib\Connection.js:82:28)
[2017-07-21 11:15:03.452] [ERROR] dateFileLog - ReferenceError: SatTimeWarningAID is not defined
    at success (C:\NB\tm.FGEMS\bizServer\biz\auto\AlarmOutputLower.js:133:65)
    at Object.exports.safeSuccess (C:\NB\yujiang.Foil.Node\src\yjError.js:98:13)
    at Object.exports.handleResult (C:\NB\yujiang.Foil.Node\src\yjError.js:154:12)
    at Query._callback (C:\NB\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:202:38)
    at Query.Sequence.end (C:\NB\node_modules\mysql\lib\protocol\sequences\Sequence.js:96:24)
    at Query._handleFinalResultPacket (C:\NB\node_modules\mysql\lib\protocol\sequences\Query.js:143:8)
    at Query.EofPacket (C:\NB\node_modules\mysql\lib\protocol\sequences\Query.js:127:8)
    at Protocol._parsePacket (C:\NB\node_modules\mysql\lib\protocol\Protocol.js:271:23)
    at Parser.write (C:\NB\node_modules\mysql\lib\protocol\Parser.js:77:12)
    at Protocol.write (C:\NB\node_modules\mysql\lib\protocol\Protocol.js:39:16)
[2017-07-21 11:18:03.566] [ERROR] dateFileLog - ReferenceError: SatTimeWarningAID is not defined
    at success (C:\NB\tm.FGEMS\bizServer\biz\auto\AlarmOutputLower.js:82:65)
    at Object.exports.safeSuccess (C:\NB\yujiang.Foil.Node\src\yjError.js:98:13)
    at Object.exports.handleResult (C:\NB\yujiang.Foil.Node\src\yjError.js:154:12)
    at Query._callback (C:\NB\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:202:38)
    at Query.Sequence.end (C:\NB\node_modules\mysql\lib\protocol\sequences\Sequence.js:96:24)
    at Query._handleFinalResultPacket (C:\NB\node_modules\mysql\lib\protocol\sequences\Query.js:143:8)
    at Query.EofPacket (C:\NB\node_modules\mysql\lib\protocol\sequences\Query.js:127:8)
    at Protocol._parsePacket (C:\NB\node_modules\mysql\lib\protocol\Protocol.js:271:23)
    at Parser.write (C:\NB\node_modules\mysql\lib\protocol\Parser.js:77:12)
    at Protocol.write (C:\NB\node_modules\mysql\lib\protocol\Protocol.js:39:16)
[2017-07-21 13:53:53.905] [ERROR] dateFileLog - { Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '' at line 1
    at Query.Sequence._packetToError (C:\NB\node_modules\mysql\lib\protocol\sequences\Sequence.js:48:14)
    at Query.ErrorPacket (C:\NB\node_modules\mysql\lib\protocol\sequences\Query.js:82:18)
    at Protocol._parsePacket (C:\NB\node_modules\mysql\lib\protocol\Protocol.js:271:23)
    at Parser.write (C:\NB\node_modules\mysql\lib\protocol\Parser.js:77:12)
    at Protocol.write (C:\NB\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (C:\NB\node_modules\mysql\lib\Connection.js:82:28)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    --------------------
    at Protocol._enqueue (C:\NB\node_modules\mysql\lib\protocol\Protocol.js:135:48)
    at PoolConnection.Connection.query (C:\NB\node_modules\mysql\lib\Connection.js:185:25)
    at doQuery (C:\NB\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:128:25)
    at success (C:\NB\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:119:13)
    at Object.exports.safeSuccess (C:\NB\yujiang.Foil.Node\src\yjError.js:98:13)
    at Object.exports.handleResult (C:\NB\yujiang.Foil.Node\src\yjError.js:154:12)
    at C:\NB\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:57:11
    at Ping.onPing (C:\NB\node_modules\mysql\lib\Pool.js:94:5)
    at bound (domain.js:280:14)
    at Ping.runBound [as _callback] (domain.js:293:12)
  code: 'ER_PARSE_ERROR',
  errno: 1064,
  sqlState: '42000',
  index: 0 }
[2017-07-21 13:53:53.911] [ERROR] dateFileLog - Error: ER_PARSE_ERROR: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '' at line 1
    at Request.<anonymous> (C:\NB\yujiang.Foil.Node\src\yjREST.engine.remote.restler.js:43:33)
    at emitTwo (events.js:106:13)
    at Request.emit (events.js:191:7)
    at Request._fireSuccess (C:\NB\node_modules\restler\lib\restler.js:220:12)
    at C:\NB\node_modules\restler\lib\restler.js:158:20
    at IncomingMessage.auto (C:\NB\node_modules\restler\lib\restler.js:394:7)
    at Request._encode (C:\NB\node_modules\restler\lib\restler.js:195:29)
    at C:\NB\node_modules\restler\lib\restler.js:154:16
    at Request._decode (C:\NB\node_modules\restler\lib\restler.js:170:7)
    at IncomingMessage.<anonymous> (C:\NB\node_modules\restler\lib\restler.js:147:14)
[2017-07-21 14:44:09.829] [ERROR] dateFileLog - TypeError: Cannot read property 'WarningAID' of undefined
    at C:\NB\tm.FGEMS\bizServer\biz\warning\getDeviceAbnormal.{m}.js:65:51
    at Object.async.whilst (C:\NB\node_modules\async\lib\async.js:678:13)
    at success (C:\NB\tm.FGEMS\bizServer\biz\warning\getDeviceAbnormal.{m}.js:37:19)
    at Object.exports.safeSuccess (C:\NB\yujiang.Foil.Node\src\yjError.js:98:13)
    at Object.exports.handleResult (C:\NB\yujiang.Foil.Node\src\yjError.js:154:12)
    at Query.<anonymous> (C:\NB\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:202:38)
    at bound (domain.js:280:14)
    at Query.runBound [as _callback] (domain.js:293:12)
    at Query.Sequence.end (C:\NB\node_modules\mysql\lib\protocol\sequences\Sequence.js:96:24)
    at Query._handleFinalResultPacket (C:\NB\node_modules\mysql\lib\protocol\sequences\Query.js:143:8)
[2017-07-21 14:44:09.839] [ERROR] dateFileLog - Error: Cannot read property 'WarningAID' of undefined
    at Request.<anonymous> (C:\NB\yujiang.Foil.Node\src\yjREST.engine.remote.restler.js:43:33)
    at emitTwo (events.js:106:13)
    at Request.emit (events.js:191:7)
    at Request._fireSuccess (C:\NB\node_modules\restler\lib\restler.js:220:12)
    at C:\NB\node_modules\restler\lib\restler.js:158:20
    at IncomingMessage.auto (C:\NB\node_modules\restler\lib\restler.js:394:7)
    at Request._encode (C:\NB\node_modules\restler\lib\restler.js:195:29)
    at C:\NB\node_modules\restler\lib\restler.js:154:16
    at Request._decode (C:\NB\node_modules\restler\lib\restler.js:170:7)
    at IncomingMessage.<anonymous> (C:\NB\node_modules\restler\lib\restler.js:147:14)
[2017-07-21 14:44:10.674] [ERROR] dateFileLog - TypeError: Cannot read property 'WarningAID' of undefined
    at C:\NB\tm.FGEMS\bizServer\biz\warning\getDeviceAbnormal.{m}.js:65:51
    at Object.async.whilst (C:\NB\node_modules\async\lib\async.js:678:13)
    at success (C:\NB\tm.FGEMS\bizServer\biz\warning\getDeviceAbnormal.{m}.js:37:19)
    at Object.exports.safeSuccess (C:\NB\yujiang.Foil.Node\src\yjError.js:98:13)
    at Object.exports.handleResult (C:\NB\yujiang.Foil.Node\src\yjError.js:154:12)
    at Query.<anonymous> (C:\NB\yujiang.Foil.Node\src\yjDBService.engine.mysql.js:202:38)
    at bound (domain.js:280:14)
    at Query.runBound [as _callback] (domain.js:293:12)
    at Query.Sequence.end (C:\NB\node_modules\mysql\lib\protocol\sequences\Sequence.js:96:24)
    at Query._handleFinalResultPacket (C:\NB\node_modules\mysql\lib\protocol\sequences\Query.js:143:8)
[2017-07-21 14:44:10.681] [ERROR] dateFileLog - Error: Cannot read property 'WarningAID' of undefined
    at Request.<anonymous> (C:\NB\yujiang.Foil.Node\src\yjREST.engine.remote.restler.js:43:33)
    at emitTwo (events.js:106:13)
    at Request.emit (events.js:191:7)
    at Request._fireSuccess (C:\NB\node_modules\restler\lib\restler.js:220:12)
    at C:\NB\node_modules\restler\lib\restler.js:158:20
    at IncomingMessage.auto (C:\NB\node_modules\restler\lib\restler.js:394:7)
    at Request._encode (C:\NB\node_modules\restler\lib\restler.js:195:29)
    at C:\NB\node_modules\restler\lib\restler.js:154:16
    at Request._decode (C:\NB\node_modules\restler\lib\restler.js:170:7)
    at IncomingMessage.<anonymous> (C:\NB\node_modules\restler\lib\restler.js:147:14)
[2017-07-21 16:14:13.898] [ERROR] dateFileLog - C:\NB\tm.FGEMS\bizServer\biz\auto\AlarmOutputLower.js:197
function  iniFormaterrLocalTimeNowStr(){
^^^^^^^^
SyntaxError: Unexpected token function
    at createScript (vm.js:56:10)
    at Object.runInThisContext (vm.js:97:10)
    at Module._compile (module.js:542:28)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
    at require (internal/module.js:20:19)
    at Object.foundFile (C:\NB\yujiang.Foil.Node\src\yjApp.js:192:7)
